### PR TITLE
Onboarding openshift/ols-load-generator config to the CI

### PR DESCRIPTION
**Pull request by vishnuchalla**
### Description
Onboarding openshift/ols-load-generator repo configuration to the CI.

### Testing
Will test and verify through the PR.

### CODE REVIEW

**Comment by vishnuchalla** [comment]
cc: @bparees @xrajesh

File: ci-operator/config/openshift/ols-load-generator/OWNERS [hunk 1]
```diff
@@ -0,0 +1,9 @@
+approvers:
+- bparees
+- xrajesh
+- eparis
+options: {}
+reviewers:
+- bparees
+- xrajesh
+- eparis
```
Comment (xrajesh) [inline ci-operator/config/openshift/ols-load-generator/OWNERS:9]: Do we need eparis as reviewer and approver - I dont think so .  Could we add @tisnik as approver and reviewer. 
Can add @raptorsun as a reviewer.

Comment (vishnuchalla) [reply ci-operator/config/openshift/ols-load-generator/OWNERS]: Updated the list as suggested. Keeping `eparis` only as an approver, if that sounds okay.

**Comment by vishnuchalla** [comment]
@xrajesh is this PR good to go?

File: ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
```
Comment (xrajesh) [inline ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml:5]: @vishnuchalla may be we want to update to latest version ?

Comment (vishnuchalla) [reply ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml]: Done

**Comment by vishnuchalla** [comment]
@xrajesh @tisnik just a gentle reminder again.

**Pull request merged**